Makefile

```diff
@@ -86,9 +86,19 @@ optimize_assets: optimize_svg
 lint_optimized_assets: optimize_assets
 	(! git diff --name-only | grep "\.svg$$") || (echo "Error: Optimize assets using 'make optimize_assets'"; exit 1)
 
-update_country_dialing_codes:
-	bundle exec ./scripts/pinpoint-supported-countries > config/country_dialing_codes.yml
-	yarn normalize-yaml config/country_dialing_codes.yml
+update_pinpoint_supported_countries:
+	bundle exec ./scripts/pinpoint-supported-countries > config/pinpoint_supported_countries.yml
+	bundle exec ./scripts/deep-merge-yaml \
+		--comment 'Generated from `make update_pinpoint_supported_countries`' \
+		--sources \
+		-- \
+		config/pinpoint_supported_countries.yml \
+		config/pinpoint_overrides.yml \
+		> config/country_dialing_codes.yml
+	yarn normalize-yaml config/country_dialing_codes.yml config/pinpoint_supported_countries.yml
```

Comment:
@mitchellhenke updated to merge into `config/pinpoint_supported_countries.yml` and check via lint, PTAL!

Reply:
Logic looks good to me! I think we may need to add some more overrides since `config/country_dialing_codes.yml` is deleting a bunch of countries?

Reply:
Done in bcf0055e27ee517f93960dbb266b41e589359d37 and d7cde96760ccbd7fb06c2d66cdabfd697188de02